perf/ftdc_rollups.go

```diff
@@ -71,6 +71,11 @@ func CreatePerformanceStats(dx *ftdc.ChunkIterator) (*PerformanceStatistics, err
 					perfStats.timers.extractedDurations,
 					extractValues(convertToFloats(metric.Values), lastValue)...,
 				)
+				// In order to avoid memory panics, reject
+				// anything larger than 4GB.
+				if len(perfStats.timers.extractedDurations) > 500000000 {
+					return nil, errors.New("size of ftdc file exceeds 4GB")
```

Comment:
if we return an error here won't we be wasting time on the next two attempts?

Reply:
Yeah, I think its fine for it to run twice more since there are not many jobs this large and this is a temporary solution.